tests/framework/unit_tests/TSA/testARMA.py

```diff
@@ -265,6 +263,25 @@ def createARMASignal(slags, nlags, pivot, noise=None, intercept=0, plot=False):
 checkArray('Simple ARMA AR', okay_ar, check['ar'], float, tol=1e-3)
 checkArray('Simple ARMA MA', okay_ma, check['ma'], float, tol=1e-3)
 checkFloat('Simple ARMA variance', 0.9532563046953576, check['var'], tol=1e-3)
+# residual
```

Comment:
Is it worth leaving the commented code in the commit history?

Reply:
I think we're going to need to address this particular unsolved content sooner than later, and this provides a good starting place for exploration I hope. I think it's worthwhile to leave, let me know and we can discuss it further.

Reply:
Sounds good to me.